src/scripts/itemDiscuss/dimItemDiscuss.directive.html

```diff
@@ -79,4 +79,4 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
we should be consistent with end of line new lines between js + css + html

the `.editorconfig` file should set that for your depending on your IDE